main.c

```diff
@@ -249,10 +249,7 @@ bool run_code_py(safe_mode_t safe_mode) {
         serial_write("\n");
         if (autoreload_is_enabled()) {
             serial_write_compressed(translate("Auto-reload is on. Simply save files over USB to run them or enter REPL to disable.\n"));
-        } else if (safe_mode != NO_SAFE_MODE) {
-            serial_write_compressed(translate("Running in safe mode! "));
-            serial_write_compressed(translate("Auto-reload is off.\n"));
-        } else if (!autoreload_is_enabled()) {
```

Comment:
Why are you deleting this?

Reply:
It gets printed at line 266. https://github.com/adafruit/circuitpython/blob/d01b9ce933048c69c7611cd4f512baa040f2f101/main.c#L266-L271

Reply:
It isn't because of `serial_connected_at_start`. It'd probably be clearer to factor out these messages into a function and call it at the start if serial is connected or while we are waiting for a serial connection.

Reply:
I did think about creating a function for the following: _(it gets called at 2 places)_

```c
if (autoreload_is_enabled()) {
    serial_write_compressed(translate("Auto-reload is on. Simply save files over USB to run them or enter REPL to disable.\n"));
} else {
    serial_write_compressed(translate("Auto-reload is off.\n"));
}
if (safe_mode != NO_SAFE_MODE) {
    serial_write_compressed(translate("Running in safe mode! "));
    serial_write_compressed(translate("Not running saved code.\n"));
}
``` 

I am not sure how `serial_connected_at_start` is preventing line 266 to 271..... from running.